platforms/ide/problems-api/src/main/java/org/gradle/api/problems/internal/DefaultProblemBuilder.java

```diff
@@ -17,19 +17,24 @@
 package org.gradle.api.problems.internal;
 
 import com.google.common.collect.ImmutableList;
-import com.google.common.collect.ImmutableMap;
+import org.gradle.api.Action;
 import org.gradle.api.problems.ProblemGroup;
 import org.gradle.api.problems.ProblemId;
 import org.gradle.api.problems.Severity;
 import org.gradle.api.problems.SharedProblemGroup;
 
 import javax.annotation.Nullable;
 import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
-import java.util.Map;
 
-public class DefaultProblemBuilder implements InternalProblemBuilder {
+public  class DefaultProblemBuilder implements InternalProblemBuilder {
```

Comment:
❓ Is there a spec for this change to help understand the reasoning?

Reply:
I considered this as an internal refactoring, so I don't have any spec for this. At the same time, I'll add some description very soon to this PR.

Reply:
I've updated the PR's description to reflect to your question.